microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -1107,49 +1006,49 @@ See also:
 * [Application tool, to manage storage](../storage/application-tool.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 128.
-* This checkbox is present in Microsoft Edge Stable 125.
-
-
-<!-- ====================================================================== -->
-## Enable Speculative Loads Panel in Application panel
-<!-- checkbox has no (?) link pointing to this anchor wording -->
-
-_new as of Microsoft Edge 121_
-
-See the above section, showing the newer label for the checkbox.
+* This checkbox is present in Microsoft Edge Canary 129.
+* This checkbox is present in Microsoft Edge Stable 127.
 
 
 <!-- ====================================================================== -->
-## Enable Storage Buckets Tree in Application panel
-<!-- checkbox has no (?) link pointing to this anchor wording -->
+## Performance panel: Enable annotations
 
-_new as of Microsoft Edge 118_
+_new as of Microsoft Edge Canary 129_
 
-Whether to enable the Storage Buckets Tree in the **Application** tool.
-<!-- todo old: need text -->
+Enables annotations in the Performance tool.
+<!-- todo: need desc -->
```

Comment:
need desc of **Performance panel: Enable annotations** experiment checkbox - what kind of annotations? context?
@zoherghadyali @leahmsft @captainbrosset

Reply:
not sure if related at all:
125 introduced checkbox w/ label:
"Enable save and load trace with **annotations** in **Performance** Panel"
Canary 128 changed that checkbox label to:
"Enable saving and loading traces with _modifications_ in the **Performance** panel"
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/#enable-save-and-load-trace-with-annotations-in-performance-panel